composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require-dev": {
         "csa/guzzle-cache-middleware": "^1.0",
-        "elife/api": "dev-master",
+        "elife/api": "dev-feature/subject-content",
```

Comment:
Doesn't work as that hasn't compiled the schema yet, tests pass as it's a new property and the schema is currently unaware of it.